.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -125,6 +125,16 @@ runs:
         wellKnownPolicies:
           ebsCSIController: true
       EOF
+  - name: tag oidc provider of the cluster
+    shell: bash
+    run: |
+      for arn in $(aws iam list-open-id-connect-providers --query "OpenIDConnectProviderList[*].{ARN:Arn}" --output text); do
```

Comment:
Nice! Can we tag it with the github run url like we do in the clusterConfig so that these tags are consistent?